images/kcp-upgrade/upgrade.sh

```diff
@@ -38,12 +38,12 @@ if [ "$current_kcp_version" != "$latest_kcp_version" ]; then
   sed -i "s,$current_kcp_version,$latest_kcp_version,g" .github/workflows/build-push-images.yaml
   sed -i "s,$current_kcp_version,$latest_kcp_version,g" .github/workflows/local-dev-ci.yaml
   sed -i "s,$current_kcp_version,$latest_kcp_version,g" DEPENDENCIES.md
-  sed -i "s,$current_kcp_version,$latest_kcp_version,g" ckcp/openshift/overlays/dev/kustomization.yaml
+  sed -i "s,$current_kcp_version,$latest_kcp_version,g" developer/ckcp/openshift/overlays/dev/kustomization.yaml
   sed -i "s,$current_kcp_version,$latest_kcp_version,g" docs/kcp-registration.md
```

Comment:
path needs to be updated to architect/docs/kcp-registration.md

Reply:
Done (but it's going to move again soon :wink: )